api/v1alpha/subschema/sauce.schema.json

```diff
@@ -63,6 +63,16 @@
           "type": "integer",
           "minimum": 0
         },
+        "maxAttempt": {
+          "description": "The maximum number of attempts to run a suite.",
+          "type": "integer",
+          "minimum": 1
+        },
+        "minPass": {
+          "description": "The threshold of passed attempts.",
+          "type": "integer",
+          "minimum": 0
```

Comment:
[Duplicate] Shouldn't this value to be `1` ?
Otherwise that means that the suite can pass when no attempts actually passed.